common/src/main/java/com/wynntils/handlers/item/ItemHandler.java

```diff
@@ -248,6 +251,27 @@ private ItemAnnotation calculateAnnotation(ItemStack itemStack, StyledText name)
         return annotation;
     }
 
+    private StyledText simplifyName(StyledText name) {
+        String str = name.getString();
+
+        final Pattern[] PATTERNS = {
+            Pattern.compile("^§6(?:Buying|Selling) [^ ]+ (.+)§6 for .+ Each$"),
+            Pattern.compile("^§6(?:Buying|Selling) [^ ]+ (.+)$"),
+            Pattern.compile("^§7§l[^ ]+x (.+)$")
```

Comment:
isn't `[^ ]` supposed to be a number? I think `\\d` should work fine.

Reply:
The problem is \\d doesn't match commas. You could write (\\d,\\d|\\d) but that only goes up to a million and [^ ] is easier to write and does the same thing. Although I suppose [0-9,] would also work.

Reply:
I missed this resolved conversation before. If you really are matching a number, do show this explicitly. `[\\d,]+` would be much better.